### PR TITLE
bug fix for auth signature

### PR DESCRIPTION
**Pull request by rocaltair**
* signature fix for empty params for do_http_request
* add signature debug logs
* add test case for empty params

### CODE REVIEW

File: fc2/client.py [hunk 1]
```diff
@@ -79,7 +79,10 @@ def _build_common_headers(self, method, path, customHeaders = {}, unescaped_quer
         return headers
 
     def do_http_request(self, method, serviceName, functionName, path, headers={}, params=None, body=None):
```
Comment (rsonghuster) [inline fc2/client.py]: 这个params的默认参数为什么不直接是{}

File: fc2/auth.py [hunk 2]
```diff
@@ -28,10 +29,11 @@ def sign_request(self, method, unescaped_path, headers, unescaped_queries=None):
         date = headers.get('date', '')
         canonical_headers = Auth._build_canonical_headers(headers)
         canonical_resource = unescaped_path
-        if unescaped_queries:
+        if unescaped_queries != None:
             canonical_resource = Auth._get_sign_resource(unescaped_path, unescaped_queries)
         string_to_sign = '\n'.join(
             [method.upper(), content_md5, content_type, date, canonical_headers + canonical_resource])
+        logging.debug('string to sign:{0}'.format(string_to_sign))
```
[the remark this answers is not in the document]
Comment (rocaltair) [reply fc2/auth.py]: 不会，可以参考一下其他的几个文件
